README.md

```diff
@@ -2,7 +2,7 @@
   
 # RustWorks
 
-  <i>Creating a truly free operating system for calculators</i>  
+  <i>Creating a truly free operating system for calculators.</i>  
```

Comment:
If I remember correctly, italics can be done using markdown `*like this*` (shows up *like this*).